<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UE Template Packager</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_root" stretch="0,0,0,0,0,0,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="grpProfile">
      <property name="title">
       <string>Profil</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_profile">
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbProfile"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnNewProfile">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btnSaveProfile">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblProfile">
         <property name="text">
          <string>Active profile :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnRenameProfile">
         <property name="text">
          <string>Rename</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpProject">
      <property name="title">
       <string>Projet</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_project">
       <item row="0" column="0">
        <widget class="QLabel" name="lblTemplate">
         <property name="text">
          <string>UE5 Project Folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edTemplate">
         <property name="placeholderText">
          <string>/path/to/my/ue_project</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnBrowseTemplate">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblOut">
         <property name="text">
          <string>Output folder:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edOut">
         <property name="placeholderText">
          <string>/path/to/output</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnBrowseOut">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblPattern">
         <property name="text">
          <string>ZIP Naming:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="edPattern">
         <property name="toolTip">
          <string>{project} = nom du projet, {ueversion} = 5.x, {date} = AAAAMMJJ_HHMMSS</string>
         </property>
         <property name="text">
          <string>{project}_UE{ueversion}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpPlugins">
      <property name="title">
       <string>Plugins</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Select plugins to exclude in the ZIP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listPlugins"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpVersions">
      <property name="title">
       <string>UE Versions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_versions">
       <item row="1" column="0" colspan="2">
        <widget class="QListView" name="listVersions"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblVersions">
         <property name="text">
          <string>Select the versions to package:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnBuild">
        <property name="text">
         <string>Zip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_actions">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenOut">
        <property name="text">
         <string>Open Output Folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vLayout_feedback">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txtLogs">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="footerFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <layout class="QHBoxLayout" name="hLayout_footer">
       <item>
        <spacer name="spacer_footer_left">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblFooter">
         <property name="text">
          <string>Copyright Â© SCHARTIER Isaac trading as Somndus Studio</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_footer_right">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuProfiles">
    <property name="title">
     <string>Profiles</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProfiles"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
